370-auth-ui/web-ui.md

```diff
@@ -0,0 +1,183 @@
+
+```
+|---------------------------|
+| Login with Google         |
+|---------------------------|
+| Login with Facebook       |
+|---------------------------|
+
+              Or
+
+|--------------------------------------|  |----------|
+| Enter in your email or username here |  | Continue |
+|--------------------------------------|  |----------|
+
+Login with a phone number instead.
+```
+
+If the user choose to sign in with an IdP, the user is redirected to the authorization endpoint of that IdP. If the user continues with a login ID, the user is directed to the password page.
+
+Separating the login ID and the password into two pages allow future improvements, such as [Open ID Provider Issuer Discovery](https://openid.net/specs/openid-connect-discovery-1_0.html#IssuerDiscovery). For example, the user can type in their Google account name and then be redirected to the authorization endpoint of Google.
+
+### The sign in password page
+
+The sign in password page displays a visibility toggleable password field *WITHOUT* password requirements.
+
+```
+|------------------------|  |----------|
+| Create a password here |  | Continue |
+|------------------------|  |----------|
+```
+
+### The sign up page
+
+The sign up page displays the first login ID key default. Other login ID keys are available to choose.
+
+```
+Sign up with email
+|--------------------------|  |----------|
+| Enter in your email here |  | Continue |
+|--------------------------|  |----------|
+
+Sign up with phone instead.
+Sign up with username instead.
+```
+
+### The sign up password page
+
+The sign up password page displays a visibility toggleable password field with password requirements.
+
+```
+|------------------------|  |----------|
+| Create a password here |  | Continue |
+|------------------------|  |----------|
+
+- [ ] At least one digit
+- [ ] At least one uppercase English character
+- [ ] At least one lowercase English character
+- [ ] At least one symbols ~`!@#$%^&*()-_=+[{]}\|;:'",<.>/?
+- [ ] At least 8 characters long
+```
+
+### The settings endpoint
+
+The settings endpoint `/settings` allows the user to manage their account. Features include but not limited to:
+
+- Manage sessions at Skygear Auth
+- Manage OAuth authorizations
+- Manage OAuth grants
+- Update metadata
+- Change password
+- Configure multi-factor authentication
+- Manage identities (e.g. login IDs and connections with external IdPs)
+
+### The verification endpoint
+
+The verification endpoint `/verify` allows the user to verify their login ID. This page requires authenticated user. Unauthenticated access is redirected to the root endpoint with `redirect_uri=/verify`. Verification is not a strict requirement. For example, if auto send verification email is enabled, the email is sent behind the scene and the user is redirected to the application immediately after performing the Authorization Code Flow successfully.
```

Comment:
Would like to know how user do verification for native app? User will complete the authorization flow without the app, and user will need to login again outside the app in browser?